lib/pengine/common.c

```diff
@@ -285,11 +285,13 @@ static pcmk__cluster_option_t pe_opts[] = {
 void
 pe_metadata(void)
 {
-    pcmk__print_option_metadata("pacemaker-schedulerd",
-                                "Pacemaker scheduler options",
-                                "Cluster options used by Pacemaker's scheduler"
-                                    " (formerly called pengine)",
-                                pe_opts, PCMK__NELEM(pe_opts));
+    char *s = pcmk__format_option_metadata("pacemaker-schedulerd",
+                                           "Pacemaker scheduler options",
+                                           "Cluster options used by Pacemaker's scheduler"
+                                               " (formerly called pengine)",
```

Comment:
Separately, I think enough time has gone by to drop the formerly called bit for the controller and scheduler ...

Reply:
Added a new commit on top.